### DISCUSSION TITLE
Error during boot

### DISCUSSION THREAD

**Discussion by kepello** [thehighnibble/thehighnibble.github.io#158] [IMSAI8080esp replica]
When connecting power to the ESP32, with my laptop connected to the USB cable for terminal, I see the following errors.  I located a previous post about the same error message [https://github.com/orgs/thehighnibble/discussions/86](here).  

The suggestion was to check the uSD Card.  The card appears correct when connected to my laptop, with 3 folders (cromemco, imsai and www).  

The individual replied that they suspected they needed to reprogram the ESP32, but there was no comment made regarding the outcome.  I don't know how to reprogram the ESP32 if that is the problem.  Help?

          rst:0x3 (SW_RESET),boot:0x3f (SPI_FAST_FLASH_BOOT)
          configsip: 188777542, SPIWP:0xee
          clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
          mode:DIO, clock div:1
          load:0x3fff0030,len:4940
          ho 0 tail 12 room 4
          load:0x40078000,len:15120
          load:0x40080400,len:3764
          entry 0x40080620
          E (57) flash_parts: partition 0 invalid magic number 0x10aa
          E (58) boot: Failed to verify partition table
          E (59) boot: load partition table error!
          ets Jun  8 2016 00:22:57

**Comment by MKoenen1** [comment]
It looks like your ESP32 is not communicating with the psram.  Did you solder the 6 extra pins on the ESP32?  Check your connections on the psram smd.  Take a look at the previous discussion #157.  It has a good boot log in the psram init.

**Comment by kepello** [comment]
I found and followed the steps to re-program the ESP32, and now it boots and I can access the webpage via Wifi.

If anyone else has this problem, that seems to be the required step.
